README.md

```diff
@@ -12,7 +12,7 @@ This is the home of the Holochain Rust library, being rewritten from [Go](https:
 **[Code Status:](https://github.com/holochain/holochain-rust/milestones?direction=asc&sort=completeness&state=all)** Rust version is currently Pre-Alpha. Not for production use. The code has not yet undergone a security audit. We expect to destructively restructure code APIs and data chains until Beta. Prototype go version was unveiled at our first hackathon (March 2017), with go version Alpha 0 was released October 2017.  Alpha 1 was released May 2018.  We expect a developer pre-release of this Rust re-write in mid October 2018.
 <br/>
 
-| Holochain Links: | [FAQ](https://github.com/holochain/holochain-proto/wiki/FAQ) | [Developer Docs](https://holochain.github.io/holochain-rust/) | [White Paper](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf) |
+| Holochain Links: | [FAQ](https://github.com/holochain/holochain-rust/wiki/FAQ) | [Developer Docs](https://holochain.github.io/holochain-rust/) | [White Paper](https://github.com/holochain/holochain-proto/blob/whitepaper/holochain.pdf) |
```

Comment:
won't it be: https://holochain.github.io/holochain-rust/faq.html

Reply:
@Connoropolous  Sorry, yeah, I just realized myself that a simple substitution wasn't right. Your link is close but the _FAQ.html_ base name is uppercase (as was the original); I'll change that to lowercase, if you prefer. Probably better that way.

Reply:
@Connoropolous Okay, I've pushed that change, with the FAQ in a renamed _faq.md_ file and links updated accordingly.